templates/download/server/thank-you-s390x.html

```diff
@@ -4,93 +4,45 @@
       {% if releases.latest.short_version > releases.lts.short_version %}
-      <p>
-        <a  class="p-button is-wide" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'LinuxONE server', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });" href="https://cdimage.ubuntu.com/releases/{{ releases.latest.short_version }}/release/ubuntu-{{ releases.latest.full_version }}-live-server-s390x.iso">Download Ubuntu Server {{ releases.latest.full_version }}</a>
-      </p>
+        <a  class="p-button" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'LinuxONE server', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });" href="https://cdimage.ubuntu.com/releases/{{ releases.latest.short_version }}/release/ubuntu-{{ releases.latest.full_version }}-live-server-s390x.iso">Download Ubuntu Server {{ releases.latest.full_version }}</a>
       {% endif %}
     </div>
   </div>
 </section>
 
 <section class="p-strip is-shallow">
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <div class="p-heading-icon--muted">
-        <div class="p-heading-icon__header">
-          {{
-            image(
-                url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
-                alt="",
-                width="32",
-                height="28",
-                hi_def=True,
-                loading="lazy",
-                attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
-              ) | safe
-          }}
-          <h4 class="p-heading-icon__title">eBook</h4>
-        </div>
-      </div>
-      <h3 class="p-heading--4"><a href="https://pages.ubuntu.com/Forrester_TAP_Report_2016.html">Forrester Research - get more from the cloud</a></h3>
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-3">
+      <h2 class="p-heading--2">Resources</h2>
     </div>
-
-    <div class="col-4 p-divider__block">
-      <div class="p-heading-icon--muted">
-        <div class="p-heading-icon__header">
-          {{
-            image(
-                url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
-                alt="",
-                width="32",
-                height="28",
-                hi_def=True,
-                loading="lazy",
-                attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
-              ) | safe
-          }}
-          <h4 class="p-heading-icon__title">Whitepaper</h4>
-        </div>
-      </div>
-      <h3 class="p-heading--4"><a href="http://www-01.ibm.com/marketing/iwm/dre/signup?source=stg-web&S_PKG=ov47989&S_TACT=C47305CW">IDC - next generation applications on Ubuntu for LinuxONE/z</a></h3>
+    <div class="col-3">
+      <p class="p-heading--5 p-section--shallow">eBook</p>
```

Comment:
They don’t as Lyubo mentioned in the comment above

Reply:
Got it, I was going by the design. :+1:

Reply:
@lizzochek pls don't put p-section styles onto text elements. p-sections should be parents, otherwise the margin-bottom of p-headins or any text element gets overriden by the p-section margin-bottom. This way, we loose break the vertical rhythm.